### ISSUE TITLE
get_the_author_meta() not working on author pages

### ISSUE THREAD

**Issue by gregarios** [Automattic/Co-Authors-Plus#969]
In version 3.5.12 the get_the_author_meta() doesn't show anything even though there is a meta description for the author.

Example in author.php — this only prints the administrator's info:

```
<h2><?php printf( __( '%s', 'wshg' ), get_the_author() ); ?></h2>	
<?php if ( get_the_author_meta( 'description' ) ) : ?><?php the_author_meta( 'description' ); ?><?php endif; ?>
```

**Comment by gregarios** [comment]
This is the change that causes the problem, in file /co-authors-plus.php:
![screenshot-20230731-102316-AM](https://github.com/Automattic/Co-Authors-Plus/assets/7989224/8c713fac-a2cc-48cf-aba2-c1875c3d86a1)

Commenting out this fixes it as far as I can tell:
![screenshot-20230731-102304-AM](https://github.com/Automattic/Co-Authors-Plus/assets/7989224/a9017f45-1233-439e-9c54-8933a3cd09e0)

**Comment by GaryJones** [comment]
@gregarios Can you please confirm if you meant `3.5.12` (just released) rather than `3.1.12`?

**Comment by gregarios** [comment]
> @gregarios Can you please confirm if you meant `3.5.12` (just released) rather than `3.1.12`?

Oh yes sorry I meant 3.5.12

**Comment by GaryJones** [comment]
Thanks - from your screenshot, it looks like https://github.com/Automattic/Co-Authors-Plus/pull/926 has broken things. I'm looking into it.

**Comment by GaryJones** [comment]
Fixed by #970. Released as [3.5.13](https://github.com/Automattic/Co-Authors-Plus/releases/tag/3.5.13).

**Comment by gregarios** [comment]
> Fixed by #970. Released as [3.5.13](https://github.com/Automattic/Co-Authors-Plus/releases/tag/3.5.13).

Verified fixed under 3.5.13.
Thank you!
